src/fparser/two/tests/test_symbol_table.py

```diff
@@ -279,6 +279,34 @@ def test_module_use_with_rename(f2003_parser):
     assert table.lookup("other").primitive_type == "logical"
 
 
+def test_wildcard_module_search(f2003_parser):
+    """Test the wildcard_imports method of the SymbolTable."""
+    _ = f2003_parser(
+        get_reader(
+            """\
+module my_mod
+  use other_mod, only: b
```

Comment:
Do you have a test that checks that the logic works if the shadowed intrinsic is named in a use statement. I can't remember. I'm sure you already do.

Reply:
I was about to confidently reply "yes" but it turns out that I didn't. I can't quite believe it. Anyway, that then showed that SymbolTable.lookup() wasn't checking for imported Symbols so I've updated it.

Reply:
I agree that you do now :-)